Release sage 2.1.0 (#420)
openforcefield/qca-dataset-submission: submissions/2025-05-22-OpenFF-Sage-2.1.0-Torsion-Drive-Training-Dataset-v1.0/README.md

# OpenFF Sage 2.1.0 Torsion Drive Training Dataset v1.0

## Description

> A quantum chemical (QC) dataset of single point energies and properties was curated to train sage 
> [OpenFF 2.1.0 Sage](https://github.com/openforcefield/sage-2.1.0/) torsion potentials. This QC dataset with the OpenFF
>  default level of theory, B3LYP-D3BJ/DZVP, consists of torsional profiles used to train torsion parameters. 
>  This is the complete TorsionDrive dataset including torsion drives used for training OpenFF 2.0.0 Sage, in addition 
>  to others. The following datasets were used:

Reviewer: ```suggestion
A quantum chemical (QC) dataset of torsiondrive profiles was generated at the OpenFF
 default level of theory, B3LYP-D3BJ/DZVP, and curated to train valence parameters in [OpenFF 2.1.0 Sage](https://github.com/openforcefield/sage-2.1.0/). Torsion profiles were curated from the following datasets:
```

 - [OpenFF DANCE 1 eMolecules t142 v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/7f8ed2ab6c8acc4521c8ca45ff4f587b20f0bcda/submissions/2020-06-01-DANCE-1-eMolecules-t142-selected)
 
 These combined datasets were filtered with:
 
 - `ElementFilter(allowed_elements=['H', 'C', 'N', 'O', 'S', 'P', 'F', 'Cl', 'Br'])`
 - `HydrogenBondFilter(method='baker-hubbard')`
 - `ChargeCheckFilter()`

## General Information

* Date: 2025-05-22
* Class: OpenFF TorsionDrive Dataset
* Purpose: Complete set of training TorsionDrive data for OpenFF 2.1.0 Sage
* Name: OpenFF Sage 2.1.0 Torsion Drive Training Dataset v1.0
* Number of unique molecules: 953
* Number of filtered molecules: 0
* Number of driven torsions: 1300
* Number of conformers: 974
* Number of conformers (min, mean, max): 1, 1, 3
* Molecular weight (min, mean, max): 32.04, 185.54, 503.42
* Charges: -1.0, 0.0, 1.0
* Dataset submitter: Jennifer A Clark
* Dataset curator: Pavan Behara
* Dataset generator: Simon Boothroyd, John Chodera, Trevor Gokey, Hyesu Jang, Yudong Qiu, Bryon Tjanaka

## Generation Pipeline

> * `main.ipynb`: A jupyter notebook which shows how the dataset was prepared from the input files.

Reviewer: Should there be a `main.ipynb` notebook here as well?
